Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -357,6 +357,34 @@ object LocalData {
             )
         }
 
+    /**
+     * Gets a boolean that tells weather the risk level
+     * decreased or not.
+     *
+     * @return boolean if risk status lowers from red to green
+     */
+    fun hasRiskStatusLowered(): Boolean {
```

Comment:
Since this is only used for the dialog, let name this `isUserToBeNotifiedOfLoweredRiskLevel` so no one accidentally uses this data for something else.

Reply:
Yeah, you're right,  I've changed it.